examples/audio/audio_control.rs

```diff
@@ -39,7 +39,7 @@ fn pause(keyboard_input: Res<Input<KeyCode>>, music_controller: Query<&AudioSink
 
 fn volume(keyboard_input: Res<Input<KeyCode>>, music_controller: Query<&AudioSink, With<MyMusic>>) {
     if let Ok(sink) = music_controller.get_single() {
-        if keyboard_input.just_pressed(KeyCode::Plus) {
+        if keyboard_input.just_pressed(KeyCode::Equal) {
```

Comment:
`Equal` is the new `Plus`?

Reply:
Yeah, it was backwards before :(

Reply:
In this case though I think we should actually revert this change: it was "correct but not working" previously.

Reply:
```suggestion
        if keyboard_input.just_pressed(KeyCode::Plus) {
```

Reply:
![image](https://github.com/bevyengine/bevy/assets/2290685/a2b1dc42-4902-4d88-bf9b-8803d43d6a1d)
<details><summary>From Wikimedia</summary>
<p>
By This file was derived from: KB United States.svg, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=11391326
</p>
</details> 

"Plus" would be the modification from shift+equal ; winit now refers to the unmodified keys in US layout. I believe my change is correct?

But I have an azerty keyboard so I'm open to be proven wrong.